pulp_ansible/app/modelresource.py

```diff
@@ -75,6 +76,22 @@ class Meta:
         exclude = BaseContentResource.Meta.exclude + ("signing_service",)
 
 
+class CollectionVersionMarkResource(BaseContentResource):
```

Comment:
Not sure if you _need_ to, but i would add it to the import order list at the bottom.

Reply:
done

Reply:
Thank you. Actually, after thinking about it, we need to import the marks after the collection versions. So yes, we need to. ;)